pkg/operator/ceph/disruption/machinedisruption/reconcile.go

```diff
@@ -0,0 +1,119 @@
+package machinedisruption
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/coreos/pkg/capnslog"
+	healthchecking "github.com/openshift/machine-api-operator/pkg/apis/healthchecking/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
+	cephClient "github.com/rook/rook/pkg/daemon/ceph/client"
+	cephCluster "github.com/rook/rook/pkg/operator/ceph/cluster"
+	"github.com/rook/rook/pkg/operator/ceph/disruption/controllerconfig"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	controllerName                  = "machinedisruption-controller"
+	MDBNamespace                    = "openshift-machine-api"
+	MDBCephClusterNamespaceLabelKey = "rook.io/cephClusterNamespace"
+	MDBCephClusterNameLabelKey      = "rook.io/cephClusterName"
+	MDBFencingLabelKey              = "fencegroup.rook.io/cluster"
+)
+
+var log = capnslog.NewPackageLogger("github.com/rook/rook", controllerName)
```

Comment:
s/log/logger/ so that we get consistent with Rook's code base.

Reply:
done